org.eclipse.jdt.core.manipulation/refactoring/org/eclipse/jdt/internal/corext/refactoring/surround/SurroundWithTryCatchRefactoring.java

```diff
@@ -161,7 +161,7 @@ public RefactoringStatus checkActivationBasics(CompilationUnit rootNode) throws
 		RefactoringStatus result= new RefactoringStatus();
 		fRootNode= rootNode;
 
-		fAnalyzer= new SurroundWithTryCatchAnalyzer(fCUnit, fSelection);
+		fAnalyzer= new SurroundWithTryCatchAnalyzer(fCUnit, fSelection, true);
```

Comment:
I think this needs to be a preference which jdt-ls can set to true by default and false otherwise.  I think there are scenarios where a user would want to handle the exception for the method (e.g. to log something that isn't ever supposed to happen and a similar-type exception is actually expected in subsequent code).  Removing the case forces the user have to add it manually vs deleting it if it is extraneous.